src/styles/_mixin.scss

```diff
@@ -0,0 +1,7 @@
+@mixin position {
+  position: absolute;
+  bottom: 50%;
+  left: 50%;
+  content: "";
+  height: 220px;
```

Comment:
```suggestion
  content: "";
  position: absolute;
  bottom: 50%;
  left: 50%;
  height: 220px;
```

Reply:
there is a need for mixin in case you can use it for several blocks. Move duplicates to the mixin (for ::before & ::after)